### ISSUE TITLE
[UX] Confusing UI regarding "Enable interface translation to English"

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#4520]
**Description of the bug**

This is a spin-off of an otherwise unrelated Issue.

The user interface for "Enable interface translation to English" is pretty confusing. In both, the setting on on admin/config/regional/language/edit/en and when "translating" - actually overriding.

It's unclear what "translate to English" actually means:

![screenshot-translate-en](https://user-images.githubusercontent.com/842066/89498729-19cfde00-d7bf-11ea-989e-debe6fb83301.png)

The translation interface form is even more confusing:

![another-wtf-translation](https://user-images.githubusercontent.com/842066/89498781-366c1600-d7bf-11ea-97e4-610d498730e7.png)

**Proposed solution**

Define what this setting is actually supposed to do and find proper wording - and possibly a clearer workflow - for it.

It allows to *override* default strings in English.

**Comment by klonos** [comment]
@olafgrabienski thanks for pointing http://hojtsy.hu/blog/2013-aug-20/drupal-8-multilingual-tidbits-12-english-can-now-be-translated out on Zulip 🙏 ...it helps understand things a bit better.

> ...it used to be a big problem however to handle it (then English language) as a "translatable language". That is, to do text replacement in English text to satisfy your project requirements. For example where Drupal says "Log in" and "Log out", clients may request to see "Sign in" and "Sign out".
>
> Using Drupal core alone, this was possible to achieve in Drupal 7 by configuring a "Custom English" language on the site with a language code different from "en", which then allowed you to replace text with your choosing. ...
>
> Drupal 8 provides a simple and effective solution for this. By default, if you have English configured, it is not available as a translation target. However, you are only one checkbox away from enabling that feature. Go edit English and make it available for translation. This will make English behave as any other language and from then on, you can "translate" to English as well.
>
> All strings will still fall back on the built-in English text (eg. if you also have Hungarian on the site, untranslated strings in Hungarian will fall back on Drupal built-in English text). However when English is used for display, all your overrides will be effective.

**Comment by klonos** [comment]
How about this then?:

>Allow "translating" English interface text into English alternatives

**Comment by klonos** [comment]
Here's how it could look:

![image](https://user-images.githubusercontent.com/2423362/89603766-73ddab80-d872-11ea-91c8-e4ee54361962.png)

**Comment by klonos** [comment]
![Screen Shot 2020-08-07 at 5 59 38 am](https://user-images.githubusercontent.com/2423362/89604324-b3f15e00-d873-11ea-8cff-db5beca89095.png)

**Comment by indigoxela** [comment]
What a big difference a slight change in wording makes.

**Comment by indigoxela** [comment]
One remark regarding permissions: there are two independent permissions.

- Translate content 
- Administer languages 

So the help text in the "Edit string" form needs at least a check, whether the current user has access to the language configuration form.

**Comment by olafgrabienski** [comment]
> Treat English as a translatable interface language

> English has been configured as a translatable language.

I get the idea, but English has always been "translatable" (into other languages).

**Comment by olafgrabienski** [comment]
> Treat English as a translatable interface language

@klonos I've thought about "translatable" again, and I guess it really has a different meaning than in your proposal. In my understanding, something is translatable if you can translate it *into* something, and that has always been the case with English. (What do native English speakers think about it?)

Unfortunately, I didn't find a good alternative, only a quirky one:

> Treat English as a language to be translatable into

**Comment by indigoxela** [comment]
Hm... What I liked about the field labels in the mockup above is that it's not called "translate" but "provide alternative".

OK, we're *using* the translation system for that, but it has nothing to do with "translation" as-is.

**Comment by klonos** [comment]
Yeah, naming things is hard 😅 ...usually @jenlampton has better ideas than I 🙂 

I don't like "Edit string" either TBH. "string" is a developer term.
